vendor/licenses/git_submodule/apex-tmLanguage.dep.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: apex-tmLanguage
+version: 77279761696d92deb26921210f20786bf8f3ffbc
+type: git_submodule
+homepage: https://github.com/forcedotcom/apex-tmLanguage.git
+license: other
```

Comment:
```suggestion
license: bsd-3-clause
```

Reply:
Coolio - also put a PR up to have that fixed in [the source repo](https://github.com/forcedotcom/apex-tmLanguage/pull/58).

Reply:
That's been merged now @muenzpraeger - thanks for the note on this @lildude 👍